setup.py

```diff
@@ -181,6 +182,7 @@ def run(self):
     version=get_version(),
 
     python_requires=PYTHON_REQUIRES,
+    matlab_requires=MATLAB_REQUIRES,
```

Comment:
It doesn't work that way. You need to implement Matlab check version in code where we find and install matlab engine Python package.

Somewhere in this function:

https://github.com/OpenNFT/OpenNFT/blob/e8caef4a8020b46aaeb28d06166aa7e634e3bc69/setup.py#L83